charts/argo/values.yaml

```diff
@@ -157,6 +157,8 @@ server:
   # SecurityContext to set on the server pods
   podSecurityContext: {}
   name: server
+  # either HTTP or HTTPS
+  readinessProbeScheme: HTTP
```

Comment:
As above can we change this `readinessProbe.scheme`.

Reply:
Sure.